src/locale/nl.yml

```diff
@@ -684,7 +684,7 @@ feat:
       descriptionBooked: Dit zijn de mensen die je hebt geboekt en op wie je rekent
         voor het evenement. Om hun deelname te annuleren, moeten ze contact met
         jou opnemen en je kunt ze handmatig annuleren.
-      descriptionCancelled: Deze mensen hebben hun deelname om een ​​of andere reden
+      descriptionCancelled: Deze mensen hebben hun deelname om een of andere reden
```

Comment:
Related to my other comment. We prefer not to make changes on these auto-generated files. 

```suggestion
```